docs/dev/developer-guides/hello-world.md

```diff
@@ -251,7 +249,7 @@ async changeGreeting() {
 },
 ```
 
-On the top of the `<script>` tag, you may see the parts that should be filled with the address of the deployed `Greeter` contract and the path to its ABI. Let's fill these fields in the following sections.
+On the top of the `<script>` tag, you may see the parts that should be filled with the address of the deployed `Greeter` contract and the path to its ABI. We will fill these fields in the following sections.
```

Comment:
Can you change to:

At the top of the `script` tag,  you may see the parts that should be filled with the address of the deployed `Greeter` contract and the path to its `ABI`. We will fill these fields in the following sections.